### ISSUE TITLE
Cerebriform connective tissue nevus

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10194]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: [20709429](https://pubmed.ncbi.nlm.nih.gov/20709429) The cerebriform connective tissue nevus (CCTN) is one of the most characteristic skin findings. It commonly occurs on the soles of the feet, and frequently causes problems because of pain, pruritus, infection, bleeding,exudation, odor, and walking impairment.
(....) The cerebriform connective tissue nevus is frequently observed in patients with Proteus syndrome and it is an important specific criterion for diagnosis

**Parent term (use hpo.jax.org/app)**
Abnormality of connective tissue HP:0003549

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Proteus syndrome ORPHA:744

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
added as 

HP:6000009
